src/Vernou.Swashbuckle.HttpResultsAdapter.Demo.Tests/HttpResultsAsOpenApiResponse.cs

```diff
@@ -53,6 +54,31 @@ public void TypedResult(string path, string expected)
         response.Value.Content.ShouldBeEmpty();
     }
 
+    [Theory]
+    [InlineData("ok-async", "200")]
+    [InlineData("created-async", "201")]
+    [InlineData("accepted-async", "202")]
+    [InlineData("acceptedatroute-async", "202")]
+    [InlineData("nocontent-async", "204")]
+    [InlineData("badrequest-async", "400")]
+    [InlineData("validationproblem-async", "400", Skip = "Fail")]
+    [InlineData("unauthorized-async", "401")]
+    [InlineData("notfound-async", "404")]
+    [InlineData("conflict-async", "409")]
+    [InlineData("unprocessableentity-async", "422", Skip = "Fail")]
+    public void TypedResultAsync(string path, string expected)
```

Comment:
This method is the same as `TypedResult`. I prefer to reuse a existing test and add case with `[InlineData]`.

Reply:
Ok. Then simply merge those methods. Did this in a rush and not so familiar with that unit test framework. if it can be done in one method then sure only one method.